src/styles/blocks/content.scss

```diff
@@ -0,0 +1,93 @@
+.clock {
+  position: relative;
+  width: 1em;
+  height: 1em;
+  box-sizing: border-box;
+  border: $padding-clock solid #000;
+  border-radius: 50%;
+  font-size: $size-clock;
+  text-align: center;
+
+  &__hand {
+    position: absolute;
+    margin-left: -0.02em;
+    margin-top: -0.02em;
+    height: 0.03em;
+    width: 0.03em;
+    left: 50%;
+    top: 50%;
+    background-color: #000;
+    border-radius: 50%;
+
+    &--minute::after {
+      content: "";
+      border-radius: $border-radius;
+      background-color: #000;
+      position: absolute;
+      margin-left: -0.01em;
+      height: 0.25em;
+      width: 0.03em;
+      bottom: 50%;
+      left: 50%;
+      transform-origin: bottom;
+      animation: rotate-hand 3600s linear infinite;
+    }
+
+    &--second::after {
+      content: "";
+      border-radius: $border-radius;
+      background-color: #f00;
+      position: absolute;
+      margin-left: -0.01em;
+      height: 0.35em;
+      width: 0.02em;
+      bottom: 50%;
+      left: 50%;
+      transform-origin: bottom;
+      animation: rotate-hand 60s linear infinite;
+    }
```

Comment:
to much duplication create a scss placeholder or common class

Reply:
done